week3_assign/src/Prefer/Random/randomResult.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Data from "../../assets/data";
+import Section from "../../components/section";
+import styled from "styled-components";
+import MoveBtn from "../../components/moveBtn";
+import Title from "../../components/Title";
+import BtnWapper from "../../components/BtnWrapper";
+
+const RandomResult = ({ setRandom }) => {
+  const drawRandom = () => {
+    return Data[Math.floor(Math.random() * Data.length)];
+  };
+
+  const randomResult = drawRandom();
```

Comment:
이 방법도 완전 좋고! 아래에서 `randomResult.~ `로 각 프로퍼티에 접근하고 잇는걸 그냥 프로퍼티명으로 바로 접근할 수 있는 구조분해할당 방식이 있다는 점도 기억하고있기!!_!_!
```suggestion
  const {img, name} = drawRandom();
```